src/visualization/Viewer.js

```diff
@@ -29,7 +29,7 @@ ROS3D.Viewer = function(options) {
   var intensity = options.intensity || 0.66;
   var near = options.near || 0.01;
   var far = options.far || 1000;
-  var cameraPosition = options.cameraPose || {
+  var cameraPosition = options.cameraPosition || {
```

Comment:
This is more likely an error in the docs above as this can break API for many people